app/services/encrypted_doc_storage/doc_writer.rb

```diff
@@ -29,6 +29,13 @@ def write(image: nil)
       )
     end
 
+    def write_with_data(image:, encryption_key:, name:)
```

Comment:
i guess at this point we could call this method on line 21, but that doesn't really save us much unless the code changes again (which feels relatively unlikely)

Reply:
ah, I did mean to make a note to myself to re-evaluate if we wanted to do that, I think it's worthwhile, if only for consistency?